docs/resources/software_catalog.md

```diff
@@ -0,0 +1,263 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_software_catalog Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog Software Catalog Entity resource. This can be used to create and manage entities in Datadog Software Catalog using the YAML/JSON definition.
+---
+
+# datadog_software_catalog (Resource)
+
+Provides a Datadog Software Catalog Entity resource. This can be used to create and manage entities in Datadog Software Catalog using the YAML/JSON definition.
+
+## Example Usage
+
+```terraform
+// v3 service entity 
+resource "datadog_software_catalog" "service_v3" {
+  entity = <<EOF
+apiVersion: v3
+kind: service
+metadata:
+  name: shopping-cart
+  displayName: Shopping Cart
+  inheritFrom: service:otherService
+  tags:
+    - tag:value
+  links:
+    - name: shopping-cart runbook
+      type: runbook
+      url: https://runbook/shopping-cart
+    - name: shopping-cart architecture
+      provider: gdoc
+      url: https://google.drive/shopping-cart-architecture
+      type: doc
+    - name: shopping-cart Wiki
+      provider: wiki
+      url: https://wiki/shopping-cart
+      type: doc
+    - name: shopping-cart source code
+      provider: github
+      url: http://github/shopping-cart
+      type: repo
+  contacts:
+    - name: Support Email
+      type: email
+      contact: team@shopping.com
+    - name: Support Slack
+      type: slack
+      contact: https://www.slack.com/archives/shopping-cart
+  owner: myteam
+  additionalOwners:
+    - name: opsTeam
+      type: operator
+integrations:
+  pagerduty:
+    serviceURL: https://www.pagerduty.com/service-directory/Pshopping-cart
```

Comment:
```suggestion
    serviceURL: https://www.pagerduty.com/service-directory/shopping-cart
```
This seemed like a typo, kindly ignore if not! There are multiple instances in this PR. I realize this is a generated file and the source is elsewhere.